app/uk/gov/hmrc/tariffclassificationfrontend/forms/mappings/FormMappings.scala

```diff
@@ -46,4 +46,17 @@ object FormMappings {
         Map(key -> value)
     })
 
+  def oneOf(errorKey: String = "error.required", enumeration: Enumeration): FieldMapping[String] =
```

Comment:
Is this unit tested?

Reply:
Not directly (none of the methods in FormMappings are), but it is as part of the form that uses it. Couldn't hurt to add some unit tests for the methods in here too I guess

Reply:
Added unit tests for all 3 methods now.